.github/workflows/terraform-plan-apply.yml

```diff
@@ -200,7 +200,14 @@ jobs:
           sed -i 's/{{issuerUrl}}/https:\/\/login.microsoftonline.com\/${{ env.tenant_id }}\/v2.0/' mts-trial-ui/index.html
           sed -i 's/{{trialName}}/${{ env.trial_name }}/' mts-trial-ui/index.html
 
-      # Upload to storage account (to $web container), only on merge to main
+      # Add Branding Assets
+      - name: UI component - Copy Logo File
+        run: |
+          set -o errexit
+          set -o nounset
+
+          cp ../branding_sample_trial/assets/branding/img/ndph-logo.png mts-trial-ui/assets/branding/img/
```

Comment:
You've embedded the trial deployment name in here - this needs to be a variable depending on the deployment.

Reply:
This comment is still relevant - the Terraform file should not have any references to a particular trial being deployed (including the default sample_trial) - instead should be a variable.

Reply:
right. fixed

Reply:
Dynamic path has been added by Elad